src/components/PostEditor/PostEditor.test.js

```diff
@@ -136,6 +137,7 @@ describe('PostEditor', () => {
       setIsDirty: jest.fn()
     }
 
+    // KMT: these next two blocks look the same?
     test('form in editing mode', () => {
```

Comment:
weirdddd, it should be passing editing: true to the first one, want to try that?

Reply:
I did this and tests pass:
````    test('form in editing mode', () => {
      const wrapper = shallow(<PostEditor {...props}  editing={true} />)
      expect(wrapper).toMatchSnapshot()
    })